core/commands/mount_unix.go

```diff
@@ -173,14 +175,14 @@ func Mount(node *core.IpfsNode, fsdir, nsdir string) error {
 	}
 
 	var err error
-	if err = doMount(node, fsdir, nsdir); err != nil {
+	if err = doMount(node, fsdir, nsdir, allow_other); err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func doMount(node *core.IpfsNode, fsdir, nsdir string) error {
+func doMount(node *core.IpfsNode, fsdir, nsdir string, allow_other bool) error {
```

Comment:
we have the `node` here, so let's just pull out the config here:

```go
cfg := node.Repo.Config()
allow_other := cfg.Mounts.FuseAllowOther
```
and thus avoid changing the interfaces